src/main/java/com/codessquad/qna/web/users/UsersController.java

```diff
@@ -1,61 +1,59 @@
 package com.codessquad.qna.web.users;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-
-import java.util.ArrayList;
-import java.util.List;
+import org.springframework.web.bind.annotation.*;
 
 @Controller
+@RequestMapping("users")
```

Comment:
리퀘스트 매핑에는 항상 슬래시(/) 를 붙일 것을 권장해요.
```suggestion
@RequestMapping("/users")

Reply:
넵 반영하겠습니다! 👍